web/src/test/java/org/hillview/HistogramAccuracyTest.java

```diff
@@ -203,20 +320,81 @@ public void computeAccuracyTest() {
         Assert.assertNotNull(mdSchema);
         Assert.assertNotNull(mdSchema.quantization);
 
+        HashMap<String, ArrayList<Double>> results = new HashMap();
         int iterations = 5;
         for (String col : cols) {
             ColumnQuantization quantization = mdSchema.quantization.get(col);
             Assert.assertNotNull(quantization);
 
-            if (!(quantization instanceof DoubleColumnQuantization)) {
-                continue;
-            }
-            DoubleColumnQuantization dq = (DoubleColumnQuantization) quantization;
-
             double epsilon = mdSchema.epsilon(col);
 
-            double acc = this.computeSingleColumnAccuracy(col, dq, epsilon, table, iterations);
-            System.out.println("Averaged absolute error over " + iterations + " iterations: " + acc);
+            Pair<Double, Double> res = this.computeSingleColumnAccuracy(col, quantization, epsilon, table, iterations);
+            System.out.println("Averaged absolute error over " + iterations + " iterations: " + res.first);
+
+            // for JSON parsing convenience
+            ArrayList<Double> resArr = new ArrayList();
+            resArr.add(res.first); // noise
+            resArr.add(res.second); // stdev
+
+            results.put(col, resArr);
         }
+
+        FileWriter writer = new FileWriter(histogram_results_filename);
```

Comment:
I general it's not a good idea to have tests leave stuff like this on the filesystem.
You should clean-up at the end. You can use File.deleteOnExit, for example.
If you want persistent results then we should move this code to DPBenchmarks.